WooCommerce/Classes/Analytics/WooAnalyticsEvent+ProductCreationAI.swift

```diff
@@ -32,9 +34,13 @@ extension WooAnalyticsEvent {
                               properties: [Key.value.rawValue: tone.rawValue.lowercased()])
         }
 
-        static func generateDetailsTapped(isFirstAttempt: Bool) -> WooAnalyticsEvent {
-            WooAnalyticsEvent(statName: .productCreationAIGenerateDetailsTapped,
-                              properties: [Key.isFirstAttempt.rawValue: isFirstAttempt])
+        static func generateDetailsTapped(isFirstAttempt: Bool,
+                                          features: String) -> WooAnalyticsEvent {
+            let wordCount = features.split { $0 == " " || $0.isNewline }.count
+            let properties: [String: WooAnalyticsEventPropertyType?] = [Key.isFirstAttempt.rawValue: isFirstAttempt,
+                                                                        Key.featureWordCount.rawValue: wordCount]
+            return WooAnalyticsEvent(statName: .productCreationAIGenerateDetailsTapped,
+                                     properties: properties.compactMapValues { $0 })
```

Comment:
❓ I wonder why the values have to be optional. We can replace `WooAnalyticsEventPropertyType?` with `any WooAnalyticsEventPropertyType` instead, right?

Reply:
Good question. I earlier kept the `features` as an optional parameter and I missed to update it again. 

Done in https://github.com/woocommerce/woocommerce-ios/pull/13411/commits/c5aacd4a66b370ca1b1689c2b094d92c5d33bf6f